src/core/server/saved_objects/migrations/model/model.ts

```diff
@@ -235,7 +235,26 @@ export const model = (currentState: State, resW: ResponseType<AllActionStates>):
     }
   } else if (stateP.controlState === 'LEGACY_CREATE_REINDEX_TARGET') {
     const res = resW as ExcludeRetryableEsError<ResponseType<typeof stateP.controlState>>;
-    if (Either.isRight(res)) {
+    if (Either.isLeft(res)) {
+      const left = res.left;
+      if (isLeftTypeof(left, 'index_not_yellow_timeout')) {
+        // `index_not_yellow_timeout` for the LEGACY_CREATE_REINDEX_TARGET source index:
+        // A yellow status timeout could theoretically be temporary for a busy cluster
+        // that takes a long time to allocate the primary and we retry the action to see if
+        // we get a response.
+        // If the cluster hit the low watermark for disk usage the LEGACY_CREATE_TEINDEX_TARGET action will
+        // continue to timeout and eventually lead to a failed migration.
+        // To help users diagnose the problem, we link to the docs from the final error message.
```

Comment:
```suggestion
```

We're no longer only showing docLinks in the final error message so I think this comment can be removed.